components/actionbutton/metadata/mods.md

```diff
@@ -30,4 +30,4 @@
 | `--mod-actionbutton-height` |
 | `--mod-actionbutton-icon-size` |
 | `--mod-actionbutton-min-width` |
-| `--mod-actionbutton-text-to-visual` |
\ No newline at end of file
+| `--mod-actionbutton-text-to-visual` |
```

Comment:
That's kind of funny - you added the actionbutton-label-color mod but the docs just updated the text-to-visual property